[IMP] l10n: update AR documentation

samueljlieber opened the pull request:
Task ID: [#2908400](https://www.odoo.com/web#id=2908400&cids=3&menu_id=4720&action=333&active_id=6050&model=project.task&view_type=form)

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/2430).

samueljlieber commented:
Hey @StraubCreative 😄
This doc is ready for a preliminary review. Please let me know if you have any suggestions for improvement!

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -435,205 +476,250 @@ For these transactions it’s important to have into consideration the next requ
 Reports
 =======
 
 As part of the localization the next Financial reports were added:
```
Suggestion on how to rewrite these lean introductions by providing more expository context, as well as navigation/placement information so the user doesn't need to rely on the images to make sense of what's being discussed.

```suggestion
As part of the localization installation, Financial reporting for Argentina was added inside the
:guilabel:`Accounting` dashboard. Access these reports by navigating to :menuselection:`Accounting
--> Reporting --> Argentinian Reports`
```

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -435,205 +476,250 @@ For these transactions it’s important to have into consideration the next requ
-.. image:: argentina/argentina19.png
+.. image:: argentina/sales-vat-book.png
    :align: center
+   :alt: Sales VAT book.
 
-Libro de IVA Compras
-~~~~~~~~~~~~~~~~~~~~
+The :guilabel:`Sales VAT` book report can be exported in a Zip file :guilabel:`VAT BOOK (ZIP)`
+button in the top left, which contains TXT files to upload in the AFIP portal.
```
The text and the image need to switch places here.

As well, need more context in the writing:
- what is a Sales VAT book
- where is it located / how to navigate to it?
- why should a user care about it / what's it's purpose?

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -435,205 +476,250 @@ For these transactions it’s important to have into consideration the next requ
+
+.. image:: argentina/purchases-vat-book.png
    :align: center
+   :alt: Purchases VAT book.
 
-Resumen de IVA
-~~~~~~~~~~~~~~
+The :guilabel:`Purchases VAT` book report can be exported in a Zip file :guilabel:`VAT BOOK (ZIP)`
+button in the top left, which contains TXT files to upload in the AFIP portal.
```
Same here.
Switch image and text.
Needs more written content.

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -435,205 +476,250 @@ For these transactions it’s important to have into consideration the next requ
+
+VAT Summary
+~~~~~~~~~~~
 
```
There's no written content here 🥲

gza-odoo replied:
Below you can find the missing text:
_Pivot table designed to check the monthly VAT totals. This report is for internal usage, it is not sent to AFIP._
@samueljlieber

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -435,205 +476,250 @@ For these transactions it’s important to have into consideration the next requ
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+IIBB - Sales by jurisdiction
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
```
Same here-- where's the documentation?

gza-odoo replied:
Below you can find the missing text:
_Pivot table where you can validate the gross income in each jurisdiction.
Affidavit for the corresponding taxes to pay, therefore it is not sent to the AFIP._ 
@samueljlieber

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -435,205 +476,250 @@ For these transactions it’s important to have into consideration the next requ
-IIBB - Compras  por Jurisdicción
+IIBB - Purchases by jurisdiction
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
```
Also here-- where's the documentation?

gza-odoo replied:
Below you can find the missing text:
_Pivot table where you can validate the gross purchases in each jurisdiction.
Affidavit for the corresponding taxes to pay, therefore it is not sent to the AFIP._
@samueljlieber

StraubCreative reviewed (changes requested):
Hi @samueljlieber 
Here's the first batch of changes I'd like to see. A lot of the suggestions revolve mostly around the written content itself, with only a few RST corrections, namely:
- Grammar, wording, and sentence structure
- Suggestions to improve written context (and not rely so much on images)
- Improvements around content placement i.e. text goes before images.

samueljlieber commented:
Hi @StraubCreative

Thanks for all of the info in each suggestion! I made all of the requested changes except for items that need content/information from the l10n team. I left those items unresolved and I will work with the team to get the content :)

cc: @vbe-odoo

StraubCreative commented:
Looks good @samueljlieber. Very quick work 🙂

Two errors I found which are contributing to the failed ci/doc check.

1. 
`content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:: WARNING: Anonymous hyperlink mismatch: 1 references but 0 targets.
See "backrefs" attribute for IDs.`

2.
`content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:: WARNING: image file not readable: applications/finance/accounting/fiscal_localizations/localizations/argentina/bank-account-relation-error.png`

One is a hyperlink issue (not properly formatted). The other is a failed image reference (e.g. image missing, file path wrong in RST, etc.)

Be sure to run `make html` before every push to avoid failed CI checks like this ✅

gza-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,56 +84,71 @@ Environment
+   :alt: A list of AR Localization Document Types in Odoo, as defined by AFIP.
 
 .. note::
    The complete list of Identification types defined by the AFIP is included in Odoo but only the
```
Here is the complete sentence: 
The complete list of Identification types defined by the AFIP is included in Odoo but only the common ones are active.
@vbe-odoo @samueljlieber

gza-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -133,507 +157,583 @@ types available by default:
-~~~~~~~~~~~~~~~~~~~
+Sales VAT book
+~~~~~~~~~~~~~~
+
```
Below you can find the missing text:
_In this report, all the sales are recorded, which are taken as the basis for the accounting records to determine the VAT (Tax Debit)._
@samueljlieber

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -133,507 +157,595 @@ types available by default:
+  document passed to the note.
+- the document letter should be the same than the originator document (either A or B).
+- the same currency as the source document must be used. When using a secondary currency
+  there is an exchange difference  if the currency rate is different between the emission day
```
Delete sneaky whitespace 😉 

This list btw is good example where we're right to use periods at the end of each list item since they are complete sentences that fill out the stem sentence that introduces them 🧠 
```suggestion
  there is an exchange difference if the currency rate is different between the emission day
```

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -133,507 +157,595 @@ types available by default:
-#. The FCE is rejected so the Credit Note should have the field “FCE, is Cancellation?” as True. 
-#. The Credit Note, is created with the negative amount to annulate the FCE document, 
-   in this case the field “FCE, is Cancellation?” must be empty (false). 
+#. The FCE is rejected so the :guilabel:`Credit Note` should have the field :guilabel:`FCE,
+   is Cancellation?` as *True*.
+#. The :guilabel:`Credit Note`, is created annul the FCE document, in this case the field
+   :guilabel:`FCE, is Cancellation?` must be *empty* (false).
```
Grammar/formatting:
- lowercase first letter of first word in the list since it's a continuation of stem sentence before it; and
- use a semicolon here to split contrasting statements.

```suggestion
#. the FCE is rejected so the :guilabel:`Credit Note` should have the field :guilabel:`FCE,
   is Cancellation?` as *True*; or
#. the :guilabel:`Credit Note`, is created annul the FCE document, in this case the field
   :guilabel:`FCE, is Cancellation?` must be *empty* (false).
```

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -133,507 +157,595 @@ types available by default:
 
+The :guilabel:`PDF Report` related to electronic invoices that have been validated by the AFIP
+includes a barcode at the bottom of the format which represents the CAE number. The Expiration Date
+is also displayed as it's legal requirement.
```
Wording
```suggestion
is also displayed since it's a legal requirement.
```

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -133,507 +157,595 @@ types available by default:
+For auditing and troubleshooting purposes, it's possible to obtain detailed information of an
+invoice number that has been previously sent to the AFIP. To retreive this information, go into
+:doc:`Developer Mode <../../../../general/developer_mode>`, then go to the
+:menuselection:`Accounting` menu, and then click in the button :guilabel:`Consult Invoice` in AFIP.
```
Word order.
```suggestion
:menuselection:`Accounting` menu, and then click on the :guilabel:`Consult Invoice` button in AFIP.
```

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -133,507 +157,595 @@ types available by default:
-The document number needs to be registered manually and the format is validated automatically, in
-case that the format is invalid a user error will be displayed indicating the correct format that is
-expected.
+The document number needs to be registered manually and the format is validated automatically,
+however, in case the format is invalid a user error will be displayed indicating the correct format
+that is expected.
```
Some small touchups here around tense and word choice.
```suggestion
The document number needs to be registered manually and the format will be validated automatically,
however, in case the format is invalid, a user error will pop up and indicate the correct format
that is expected.
```

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,56 +84,71 @@ Environment
+   <https://drive.google.com/file/d/17OKX2lNWd1bjUt3NxfqcCKBkBh-Xlpo-/view>`_.
+
+#. :guilabel:`Upload Certificate and Private Key (Odoo)`. Once the certificate is generated, upload
+   it to Odoo using the **pencil icon** next to the field “Certificado” and select the
```
Think this is a guilabel?
```suggestion
   it to Odoo using the :guilabel:`Pencil` icon next to the field “Certificado” and select the
```

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -133,507 +157,595 @@ types available by default:
 
-You should use the pencil that is the next to the Perception amount to edit it 
-and set the correct amount. 
+Then invoice can be validated.
```
Delete line (redundant).
```suggestion
```

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -133,507 +157,595 @@ types available by default:
 =======
 
-As part of the localization the next Financial reports were added:
+As part of the localization installation, Financial reporting for Argentina was added inside the
```
Don't think this is supposed to be capitalized?
```suggestion
As part of the localization installation, financial reporting for Argentina was added inside the
```

StraubCreative reviewed (changes requested):
Great first round of revisions everyone. The doc looks much better and more readable than where it started 😄 

Thank you @samueljlieber for the speedy updates and @gza-odoo for the content suggestions.

Here's the next batch of changes which mostly include some more grammar/formatting suggestions in trying to tighten things up. 

I think after these are implemented, we can open the PR up to the Accounting team 👍

samueljlieber commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -133,507 +157,595 @@ types available by default:
 **Invoice for a customer IVA Responsable Inscripto, prefix A**.
 
-.. image:: argentina/argentina11.png
+.. image:: argentina/prefix-a-invoice-for-customer.png
    :align: center
+   :alt: Invoice for a customer IVA Responsable Inscripto, prefix A.
 
 **Invoice for an end customer, prefix B**.
 
-.. image:: argentina/argentina12.png
+.. image:: argentina/prefix-b-invoice-for-end-customer.png
    :align: center
+   :alt: Invoice for an end customer, prefix B.
 
 **Exportation Invoice, prefix E**.
 
-.. image:: argentina/argentina13.png
+.. image:: argentina/prefix-e-exporation-invoice.png
    :align: center
+   :alt: Exportation Invoice, prefix E
```
Hi @gza-odoo! Could you please provide a description for each Document Type here to provide more context? Thank you 🙏
1. Invoice for a customer IVA Responsable Inscripto, prefix A
2. Invoice for an end customer, prefix B
3. Exportation Invoice, prefix E

vbe-odoo replied:
@gza-odoo can you help me with this one?
Thanks!
cc' @samueljlieber @StraubCreative

gza-odoo replied:
Hi @samueljlieber @vbe-odoo,
Below are my comments:
**Invoice for a customer IVA Responsable Inscripto, prefix A**: _Type of document that is going to show all the taxes in detail and customer's information._ 
**Invoice for an end customer, prefix B**: _Type of document that does not detail the taxes. They are included in the total amount._ 
**Exportation Invoice, prefix E**. _Type of document used when exporting goods that shows the incoterm._

samueljlieber commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -133,507 +157,595 @@ types available by default:
-- Fiscal position: Compras/Ventas al exterior.
-- Concepto AFIP:  Products / Definitive export of goods.
-- Exempt Taxes. 
+- Journal related to the exportation Web Service;
+- Exportation document type;
+- Fiscal position: Compras/Ventas al exterior;
+- Concepto AFIP: Products / Definitive export of goods; and
+- Exempt Taxes.
```
Hi @vbe-odoo and @gza-odoo, just confirming that this section is 100% correct - thank you! 

![When the customer selected in the Invoice has set the AFIP responsibility](https://user-images.githubusercontent.com/36018073/191799293-a33efd87-0d1a-428a-81d6-047663b2e0c9.png)

StraubCreative replied:
Quick follow up:
Please confirm the edits I made in [this comment](https://github.com/odoo/documentation/pull/2430#discussion_r977048191) above the screenshot (in green) are clear/correct, thank you 🙏 
@gza-odoo @vbe-odoo 

cc: @samueljlieber

vbe-odoo replied:
Hi, @samueljlieber so the functional description it's actually correct.
But probably I will just rephrase it a bit to make more clear:
"...When the customer selected in the Invoice is configured with an AFIP responsibility type **Cliente / Proveedor del Exterior - Ley N° 19.640**, Odoo automatically assigns the: ..."

But feel free to improve it if needed, functional wise it's correct :)

cc' @gza-odoo @StraubCreative

StraubCreative replied:
Thanks @vbe-odoo we can make that adjustment.

@samueljlieber in addition to rewording suggestion from Valentino, can you also double check to see if the typo that says `assignes` is still present?

@vbe-odoo sanity check. Are these adjustments I made to the list below ok? 👇 
I removed the nested colons (`:`) since grammatically that's not correct, and replaced them with `as`, indicating that those are the assignments for each field.

```suggestion
- Journal related to the exportation Web Service;
- Exportation document type;
- Fiscal position as **Compras/Ventas al exterior**;
- Concepto AFIP as **Products / Definitive export of goods**; and
- Exempt Taxes.
```

Please let me know if this is not correct!

cc: @gza-odoo

vbe-odoo replied:
@StraubCreative thanks for the catch, yes those are great for me 💯 !
Thanks!
cc'  @samueljlieber @gza-odoo

samueljlieber commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -133,507 +157,595 @@ types available by default:
-Once the partner is selected the document type will filled automatically, based on the AFIP document
-type:
+Once the partner is selected the document type will filled automatically, based on the AFIP
+document type:
```
Confirming we are using capitalized here moving forward: _Document Type_ is an important term that has it's own menu in Odoo.

samueljlieber commented:
Commit [88a5326](https://github.com/odoo/documentation/commit/88a532639d0b280c1bc431ff95aa30c2b879f47d) updated document per ZST's suggestions (Thank you 🙏). Remaining suggestions:
- [ Description for each Document Type](https://github.com/odoo/documentation/pull/2430#discussion_r977833925)
- [Confirmation on content](https://github.com/odoo/documentation/pull/2430#discussion_r977855379)

StraubCreative commented:
We need just two quick things here @gza-odoo @vbe-odoo 
Please see @samueljlieber comment above (click the blue links in the list which will take you to each comment describing what's needed).

After those I think we can open the PR to Accounting team 🚀

samueljlieber commented:
Hi @StraubCreative, I updated the two points in my [previous comment](https://github.com/odoo/documentation/pull/2430#issuecomment-1255399784) with the comments the team provided, thank you @vbe-odoo and @gza-odoo!

Regarding [the typo](https://github.com/odoo/documentation/pull/2430#discussion_r979047412) that says `assignes` - that is no longer present 👍

Also regarding @gza-odoo's [comment ](https://github.com/odoo/documentation/pull/2430#discussion_r980630912) I made a small change for context clarity: 
`type of document that does not detail the taxes. They are included in the total amount.` 
to
`type of document that does not detail the taxes, since
  the taxes are included in the total amount.` 
  Please let me know if this is incorrect :)

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -133,507 +155,593 @@ types available by default:
 
-.. image:: argentina/argentina11.png
-   :align: center
+- **Invoice for a customer IVA Responsable Inscripto, prefix A**: type of document that is going to
+  show all the taxes in detail and customer's information.
```
Remove nested colon (`:`), wording.
```suggestion
- **Invoice for a customer IVA Responsable Inscripto, prefix A** is the type of document that
  shows all the taxes in detail along with the customer's information.
```

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -133,507 +155,593 @@ types available by default:
 
-.. image:: argentina/argentina12.png
-   :align: center
+- **Invoice for an end customer, prefix B**: type of document that does not detail the taxes, since
+  the taxes are included in the total amount.
```
Remove nested colon (`:`)
```suggestion
- **Invoice for an end customer, prefix B** is the type of document that does not detail the taxes,
  since the taxes are included in the total amount.
```

StraubCreative commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -133,507 +155,593 @@ types available by default:
 
-.. image:: argentina/argentina13.png
-   :align: center
+- **Exportation Invoice, prefix E**: type of document used when exporting goods that shows the
+  incoterm.
```
Remove nested colon (`:`)
```suggestion
- **Exportation Invoice, prefix E** is the type of document used when exporting goods that shows
  the incoterm.
```

StraubCreative reviewed (changes requested):
@gza-odoo thank you for the content updates!

@samueljlieber nice catch with the capitalizations in the list items. Let's go one step further and remove the nested colons from the list.

After this is edited we can make the PR live for Accounting team to review 🚀

gza-odoo commented:
I'm ok with what you suggest @samueljlieber :)

samueljlieber commented:
Hi @StraubCreative, your edits have been made 👍

StraubCreative commented:
Thanks @samueljlieber for the quick changes 🙏 

I'm going to open this up to Accounting for review now.

Note:
On the next batch of change requests, let's fix the commit message to `[IMP] l10n: AR update documentation` so it's country-specific and reflects the original commit for the PR.

toaa-odoo reviewed (changes requested):
Hi everyone, congratulations for your work. That's a lot of content! :)

Apparently, the diff is too large to be rendered in GitHub, so forgive me but I couldn't suggest changes straight within GitHub. There's a lot spelling and English errors that need to be addressed before I can approve the PR. Overall comments are: be consistent in the way things are spelled. For example, you can't write Invoice (with a capital i), and then vendor bills (all in lower case). Also, if an element is visible in Odoo, use :guilabel:, always. Contractions in English are considered informal (it's -- it is). There also seems to be a mix of Spanish and English words at times ? Please keep the document strictly in English (with a Spanish translation next to it if necessary).

Here's a list of issues I spotted:

Line 22: "legal requirement in Argentine" --> "in Argentina".
Line 24: "This module includes as well the VAT..." --> "...module includes the VAT summary report as well, which is..."
Line 41: drop the "For this" as the heading is sufficient to understand, --> "Go to Apps"
Line 69: no "and" after "(290 accounts);" --> "Exento (290 accounts);"
Line 92: :guilabel:`Prueba` (Testing) or :guilabel:`Produccion` (Production)
Line 105: don't --> do not // contractions of words is considered informal in English
Line 107: "When this option is selected, a file..." // added a , (comma)
Line 108: "...generated to be used the AFIP portal..." // I think a preposition is missing after "used"
Line 115: if the link provided on that line is the "next document", then I'd suggest something like --> "follow the instructions described in this document (with the hyperlink embedded within 'this document')"
Line 119: :guilabel:`Certificado`
Line 143: "Document Types" --> "document types"
Line 152: "Identification" --> "identification" // "...included in Odoo but..." --> "...included in Odoo, but..."
Line 158: "Document Type", either write in lower case, or with :guilabel: if it's a visible element (in the whole document"
Line 170: :guilabel: Responsable Inscripto
Line 181: ... regular VAT and can have various percentages.
Line 189: ...inactive in Odoo by default (removed the comma)
Line 189: ... be sure to check if that tax is not already included as inactive.
Line 202: … some accounting transactions such as invoices and vendor bills…
Line 203: again, document types is either in lower case or guilabel
Line 204: governmental fiscal authorities
Line 207: document type is in bold this time, keep it consistent throughout the text
Line 214: does not
Line 214: in forms or documents (?)
Line 235: so there is no further configuration required (?) // It seems to me the “no” was forgotten
Line 246: journal entry (in lower case)
Line 247: issuer and receiver (lower case) // fiscal regime (?, no extra -n)
Line 253: the journal can have (lower case)
Line 254: To configure journals, go to (added , )
Line 257: sales and purchases journals (either in lower case or guilabel), but no capitalization of letters for any visible elements in Odoo
Line 258: invoices or vendor bills.
Line 259: for more detail on invoices, please refer to …
Line 262: sales or purchases
Line 274: web service (lower case)
Line 306: company (lower case) // it’s not a proper noun
Line 319: ..brings the last sequence used → unclear, displays ?
Line 370: error messages pop-up, they indicate both the issue
Line 371 & 372: If an error persists, the invoice remains in draft until the issue is resolved
Line 380: indicates if the invoice has been…
Line 390: PDF report
Line 394: following responsibility → unclear, is the responsibility of Responsable Inscripto?
Line 450: Definitive export of goods; → remove the “and”, “;” already indicates the end of the sentence and bullet-listing.
Line 471: For these transactions, it is important to
Line 473: (according to the parameter table)
Line 498: from what I understand, “emisor” is spanish for emitter. Remain consistent and spell it in English like the rest of the documentation, or make it clear it is a specific use-case (either by :guilabel: if applicable, or ** **)
Line 500: cannot be validated
Line 507: To set up the guilabel Transmission Mode, go to settings and select // (comma)
Line 527 and 528: unclear, what is originator document, and passed to the note ?
Line 532: payment date. It is possible to create a credit/debit note…
Line 539: line is unclear, scenarios related to what workflow?
Line 543: what is “annul” ?
Line 554: expiration date or guilabel
Line 555: as it is a legal requirement
Line 564: it is possible
Line 578: it is also possible to retrieve (typo here) // “as reference for any possible issues” is not clear
Line 590: switched → changed ?
Line 596: validated automatically.
Line 597: However, in case the format…
Line 616: … it only displays a warning, but still allows the vendor bill to be posted
Line 618: … and it does not allow
Line 615 to 619, don’t forget to add ; or . at the end of sentences
Line 653: exempt (lower case)
Line 710: be consistent and either write extensions like this `.txt`, or **.txt** like line 108
Line 720: to the AFIP
Line 732: Affidavit ?
Line 742: Affidavit ?

samueljlieber commented:
Hi @toaa-odoo, thank you so much for your detailed corrections! 😃

In commit [d070461](https://github.com/odoo/documentation/commit/d070461a428c10a08defb4b37dc7e5e8d8ec24ac) I updated the commit message to @StraubCreative suggestion and updated the document with most of @toaa-odoo's corrections, but I do need some assistance (@vbe-odoo & @gza-odoo) regarding the points below:

1. Can you help me identify where and when document type is visible in Odoo in this doc? I changed `document type` to be lowercase throughout the document, per @toaa-odoo's suggestion, but I am unsure of where it should be labeled as visible (:guilabel:`Document Type`)
2. Line 109: I think a preposition is missing after "used”
3. Line 216: can you clarify which (forms or documents) is correct in this sentence?
4. Line 248 and 249: can you confirm “regimen” or “regime”?
5. Line 395: following responsibility → unclear, is the responsibility of Responsable Inscripto?
6. Line 528 and 529: unclear, what is originator document, and passed to the note?
7. Line 539: line is unclear, scenarios related to what workflow?
8. Line 543: what is “annul” ?
9. Line 732: Affidavit ?
10. Line 742: Affidavit ? 

Thank you everyone ☀️

gza-odoo commented:
Hi @samueljlieber,
Below are my comments:
1) What do you mean by "labeled as visible"?
2) Yes, after "used" should go "in" (I don't know why that preposition is not here since in the original Google doc that word is there).
3) We can replace the whole sentence with what I have in the Google doc: _The information required for the document types is included by default so the user does not need to fill anything on this view:_
If that's not an option I'd pick "forms".
4) Regime
5) We mean the AFIP responsibility type, which in this case is "Responsable Inscripto".
6) I would replace the sentence with: _Use the Credit and Debit Note buttons so all the information from the invoice is transferred to the new Credit/Debit Note._ 
7) I would replace the sentence with: _When creating a Credit Note we can have two scenarios:_
8) Please replace it with: _to annulate_
9/10) Affidavit is a written statement confirmed by oath or affirmation, for use as evidence in court. From my point of view, that word fits better what we want to say. The other option is the literal translation from Spanish: sworn declaration (declaracion jurada). We're open to any other option you think makes more sense.
Please @vbe-odoo let us know your thoughts regarding these suggestions.
Best.

samueljlieber commented:
Hi @gza-odoo thank you for the additional information 🙂
In commit  19e7ed3  I updated the doc for points with a ✅ with the outstanding points are listed below:
1. What I mean by this (link!) is I need to know where the words `document type` are visible within the Odoo interface (as a title?, a button?, a field name?, etc...). We specify in the documentation with special bold formatting when something is visible in the Odoo interface.
2. ✅ 
3. ✅
4. ✅
5. @toaa-odoo does @gza-odoo's comment clarify?
6. ✅
7. ✅
8. ✅
9. @toaa-odoo does @gza-odoo's comment clarify?
10. @toaa-odoo does @gza-odoo's comment clarify?

toaa-odoo commented:
> Hi @gza-odoo thank you for the additional information slightly_smiling_face In commit [19e7ed3](https://github.com/odoo/documentation/commit/19e7ed31b65ac325e931f729876828a3d7bee4ad) I updated the doc for points with a white_check_mark with the outstanding points are listed below:
> 
> 1. What I mean by this (link!) is I need to know where the words `document type` are visible within the Odoo interface (as a title?, a button?, a field name?, etc...). We specify in the documentation with special bold formatting when something is visible in the Odoo interface.
> 2. white_check_mark
> 3. white_check_mark
> 4. white_check_mark
> 5. @toaa-odoo does @gza-odoo's comment clarify?
> 6. white_check_mark
> 7. white_check_mark
> 8. white_check_mark
> 9. @toaa-odoo does @gza-odoo's comment clarify?
> 10. @toaa-odoo does @gza-odoo's comment clarify?

Hello @samueljlieber, yes it's much clearer now. Thank you!

gza-odoo commented:
Hi @samueljlieber ,
Hope you're doing great. Below are my comments:
1) Yes, you can find "Document Types" as a menu item in the configuration menu, accounting app; also, it's a title when you click on that menu. You can find it as a field name as well.
Best.

samueljlieber commented:
Hi @gza-odoo 👋
Thank you for confirming where `Document Types` is generally found within Odoo. 

Hi @toaa-odoo 🙂
I went through and set up a test database and followed along with the AR l10n doc to find specifically where in the doc we should label the text as`:guilabel:Document Type(s)`. You can see the changes in commit 8110391. 

While following along I found a small change to line 41: Searching for modules with the text `Argentina` did not show the modules in the picture and the search text in the picture is `l10n_ar` - that had the right search results.

There is an instance where the `Identification Types` are listed in the image and referenced on the interface, so I gave that a `:guilabel:` - see lines 146, & 153-154.

There is an instance where the document type's letter is referenced in the interface, so I also gave that a `:guilabel:` - see lines 232-234.

There is an instance where the field `Document Number` is referenced, I gave that a `:guilabel:` - see line 599 - and capitalized it on line 609.

I did my best to change only where referenced/visible on the interface, I hope these changes are correct! Thank you!

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
    :align: center
+   :alt: AFIP Status.
 
 Invoice Taxes
```
```suggestion
Invoice taxes
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+     :align: center
+     :alt: Tax amount included.
 
 Special Use Cases
```
```suggestion
Special use cases
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
 Special Use Cases
 ~~~~~~~~~~~~~~~~~
 
 Invoices for Services
```
```suggestion
Invoices for services
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
    :align: center
+   :alt: Service Dates.
 
 Exportation Invoices
```
```suggestion
Exportation invoices
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-   
+   :alt: Export invoice - Incoterm.
+
 Fiscal Bond
```
```suggestion
Fiscal bond
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+  - VAT rate
 
-Electronic Credit Invoice MiPyme (FCE) 
+Electronic Credit Invoice MiPyme (FCE)
```
```suggestion
Electronic credit invoice MiPyme (FCE)
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+   :align: center
+   :alt: Invoice printed report.
 
 Troubleshooting and Auditing
```
```suggestion
Troubleshooting and auditing
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-
+   :alt: Consult the last invoice number.
 
 Vendor Bills
```
```suggestion
Vendor bills
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+that the document sequence is entered by the user using the following format:
+*Document Prefix - Letter - Document Number*.
 
 Validate Vendor Bill number in AFIP
```
```suggestion
Validate vendor bill number in AFIP
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-*********************
-This tool incorporates in the vendor bill a new "Verify on AFIP" button located 
-next to the AFIP Authorization code. 
+Validate Vendor Bills in Odoo
```
```suggestion
Validate vendor bills in Odoo
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+Special use cases
 ~~~~~~~~~~~~~~~~~
+
 Untaxed Concepts
```
```suggestion
Untaxed concepts
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
    :align: center
+   :alt: VAT exempt.
 
 Perception Taxes
```
```suggestion
Perception taxes
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
    :align: center
+   :alt: Argentinian reports.
 
 VAT Reports
```
```suggestion
VAT reports
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
 
-Resumen de IVA
-~~~~~~~~~~~~~~
+VAT Summary
```
```suggestion
VAT summary
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-usually the same address than the Company. For example: has multiple stores (fiscal locations) then
-AFIP will require that you have one AFIP POS per location: this location will be printed in the
-invoice report.
+Here are some useful fields to know when working with Web Services:
```
```suggestion
Here are some useful fields to know when working with web services:
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+properly configured.
 
-Document type assignation
+Document Type assignation
```
```suggestion
Document type assignation
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -7,62 +7,71 @@ Webinars
-  the minimal configuration needed for a company  to operate in Argentina and under the AFIP
+- **l10n_ar**: adds accounting features for the Argentinian localization, which represent the
+  minimal configuration needed for a company to operate in Argentina and under the AFIP
   (Administración Federal de Ingresos Públicos) regulations and guidelines.
```
```suggestion
  (Administración Federal de Ingresos Públicos) regulations and guidelines;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -7,62 +7,71 @@ Webinars
-  Electronic Invoice via web service, based on the AFIP regulations. 
+- **l10n_ar_reports**: adds the VAT Book report which is a legal requirement in Argentina and that
+  holds detailed VAT information of sales or purchases recorded on the journal entries. This module
+  includes the VAT summary report as well, which is used to analyze the invoice.
```
```suggestion
  includes the VAT summary report as well, which is used to analyze the invoice;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -7,62 +7,71 @@ Webinars
+  Invoices via web service, based on the AFIP regulations.
+
+The following module is optional and should be installed only if it meets a specific organization
+requirement.
```
```suggestion
requirement:
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-- Perception. Advance payment of a tax that is applied on Invoices.
-- Retention. Advance payment of a tax that is applied on payments
-- Otros.
+- :guilabel:`VAT`: this is the regular VAT and can have various percentages.
```
```suggestion
- :guilabel:`VAT`: this is the regular VAT and can have various percentages;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-- Retention. Advance payment of a tax that is applied on payments
-- Otros.
+- :guilabel:`VAT`: this is the regular VAT and can have various percentages.
+- :guilabel:`Perception`: advance payment of a tax that is applied on invoices.
```
```suggestion
- :guilabel:`Perception`: advance payment of a tax that is applied on invoices;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-- When an invoice is related to a B2B transaction, a document type "A" must be used.
-- When an invoice is related to a B2C transaction, a document type "B" must be used.
-- When an invoice is related to exportation transaction, a document type "E" must be used.
+- :guilabel:`B2B transaction`, a document type :guilabel:`A` must be used.
```
```suggestion
- :guilabel:`B2B transaction`, a document type :guilabel:`A` must be used;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-- When an invoice is related to a B2C transaction, a document type "B" must be used.
-- When an invoice is related to exportation transaction, a document type "E" must be used.
+- :guilabel:`B2B transaction`, a document type :guilabel:`A` must be used.
+- :guilabel:`B2C transaction`, a document type :guilabel:`B` must be used.
```
```suggestion
- :guilabel:`B2C transaction`, a document type :guilabel:`B` must be used;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+from:
+
+- :guilabel:`wsfev1: Electronic Invoice`: is the most common service, which is used to generate
+  invoices for document types A, B, C, M  with no detail per item.
```
```suggestion
  invoices for document types A, B, C, M  with no detail per item;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+- :guilabel:`wsbfev1: Electronic Fiscal Bond`: is for those who invoice capital goods and wish to
+  access the benefit of the Electronic Tax Bonds granted by the Ministry of Economy. For more
+  details go to: `Fiscal Bond
+  <https://www.argentina.gob.ar/acceder-un-bono-por-fabricar-bienes-de-capital>`__.
```
```suggestion
  <https://www.argentina.gob.ar/acceder-un-bono-por-fabricar-bienes-de-capital>`__;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-**Unified Book**: When AFIP POS System is Preimpresa the document types (applicable to the journal)
-with the same letter will share the same sequence.  For example:
+- :guilabel:`AFIP POS Number`: is the number configured in the AFIP to identify the operations
+  related to this AFIP POS.
```
```suggestion
  related to this AFIP POS;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+- :guilabel:`AFIP POS Address`: is the field related to the commercial address registered for the
+  POS, which is usually the same address as the company. For example, if a company has multiple
+  stores (fiscal locations) then AFIP will require the company to have one AFIP POS per location.
+  This location will be printed in the invoice report.
```
```suggestion
  This location will be printed in the invoice report;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-- Invoice: FA-A 0001-00000002.
-- Credit Note: NC-A 0001-00000003.
-- Debit Note: ND-A 0001-00000004.
+  - Invoice: FA-A 0001-00000002.
```
```suggestion
  - Invoice: FA-A 0001-00000002;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-- Credit Note: NC-A 0001-00000003.
-- Debit Note: ND-A 0001-00000004.
+  - Invoice: FA-A 0001-00000002.
+  - Credit Note: NC-A 0001-00000003.
```
```suggestion
  - Credit Note: NC-A 0001-00000003;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-This condition applies when the customer has the following AFIP Responsibility type:
+- :guilabel:`A. Tax excluded`: in this case the taxed amount needs to be clearly identified in the
+  report. This condition applies when the customer has the following AFIP Responsibility type of
+  **Responsable Inscripto**.
```
```suggestion
  **Responsable Inscripto**;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-
-.. image:: argentina/argentina15.png
-   :align: center
+  - IVA Sujeto Exento
```
```suggestion
  - IVA Sujeto Exento;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-.. image:: argentina/argentina15.png
-   :align: center
+  - IVA Sujeto Exento
+  - Consumidor Final
```
```suggestion
  - Consumidor Final;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-   :align: center
+  - IVA Sujeto Exento
+  - Consumidor Final
+  - Responsable Monotributo
```
```suggestion
  - Responsable Monotributo;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+  - IVA Sujeto Exento
+  - Consumidor Final
+  - Responsable Monotributo
+  - IVA liberado
```
```suggestion
  - IVA liberado.
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-  - Unit of measurement.
-  - Bonus.
-  - VAT rate. 
+- Currency (according to the parameter table) and invoice quotation
```
```suggestion
- Currency (according to the parameter table) and invoice quotation;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-  - Bonus.
-  - VAT rate. 
+- Currency (according to the parameter table) and invoice quotation
+- Taxes
```
```suggestion
- Taxes;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+- Zone
+- Detail each item
 
+  - Code according to the Common Nomenclator of Mercosur (NCM)
```
```suggestion
  - Code according to the Common Nomenclator of Mercosur (NCM);
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+- Detail each item
 
+  - Code according to the Common Nomenclator of Mercosur (NCM)
+  - Complete description
```
```suggestion
  - Complete description;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
 
+  - Code according to the Common Nomenclator of Mercosur (NCM)
+  - Complete description
+  - Unit Net Price
```
```suggestion
  - Unit Net Price;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+  - Code according to the Common Nomenclator of Mercosur (NCM)
+  - Complete description
+  - Unit Net Price
+  - Quantity
```
```suggestion
  - Quantity;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+  - Complete description
+  - Unit Net Price
+  - Quantity
+  - Unit of measurement
```
```suggestion
  - Unit of measurement;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+  - Unit Net Price
+  - Quantity
+  - Unit of measurement
+  - Bonus
```
```suggestion
  - Bonus;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+  - Quantity
+  - Unit of measurement
+  - Bonus
+  - VAT rate
```
```suggestion
  - VAT rate.
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+
+For these transactions it's important to consider the following requirements:
+
+- specific document types (201, 202, 206, etc)
```
```suggestion
- specific document types (201, 202, 206, etc);
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+For these transactions it's important to consider the following requirements:
+
+- specific document types (201, 202, 206, etc)
+- the emitter should be eligible by the AFIP to MiPyME transactions
```
```suggestion
- the emitter should be eligible by the AFIP to MiPyME transactions;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+
+- specific document types (201, 202, 206, etc)
+- the emitter should be eligible by the AFIP to MiPyME transactions
+- the amount should be bigger than 100,000 ARS
```
```suggestion
- the amount should be bigger than 100,000 ARS;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+- specific document types (201, 202, 206, etc)
+- the emitter should be eligible by the AFIP to MiPyME transactions
+- the amount should be bigger than 100,000 ARS
+- A bank account type CBU must be related to the emisor, otherwise the invoice cannot
```
```suggestion
- A bank account type CBU must be related to the emisor, otherwise the invoice cannot;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
+- the emitter should be eligible by the AFIP to MiPyME transactions
+- the amount should be bigger than 100,000 ARS
+- A bank account type CBU must be related to the emisor, otherwise the invoice cannot
+  be validated, having these errors messages for example:
```
```suggestion
  be validated, having an error message such as the following.
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-- Use the Credit and Debit Note buttons, so the correct reference of the originator 
-  document passed to the note. 
+- use the :guilabel:`Credit and Debit Note` buttons, so all the information from the invoice is
+  transferred to the new :guilabel:`Credit and Debit Note`.
```
```suggestion
  transferred to the new :guilabel:`Credit and Debit Note`;
```

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -75,565 +84,667 @@ Environment
-#. The Credit Note, is created with the negative amount to annulate the FCE document, 
-   in this case the field “FCE, is Cancellation?” must be empty (false). 
+#. the FCE is rejected so the :guilabel:`Credit Note` should have the field :guilabel:`FCE,
+   is Cancellation?` as *True*; or
```
```suggestion
   is Cancellation?` as *True*; or;
```

toaa-odoo reviewed (changes requested):
Thanks @samueljlieber and everyone for your work. Sorry for the delay, we were busy with the OXP. The documentation's content is much better and complete now. I spotted minor changes that remain to be made, but I promise you won't hear from me again afterward. :)

General comments are:

As per the new 'contributing' guidelines: only the first letter and proper nouns are capitalized in headings.

In English, entries in a list always end with ';' and the final entry with '.'

I noticed in screenshot content/applications/finance/accounting/fiscal_localizations/localizations/argentina/select-fiscal-package.png that Argentinian is spelled 'Argentinean localization'. Both are correct ways to spell the adjective, but if you were multiple working on this, you might have spelled it differently within the module. Check among yourselves that you indeed used the same spelling throughout.

vbe-odoo commented:
> Hi @vbe-odoo and @gza-odoo can you help me determine how we should label `Argentinian/Argentinean` throughout the document per @toaa-odoo's point:
> > I noticed in screenshot content/applications/finance/accounting/fiscal_localizations/localizations/argentina/select-fiscal-package.png that Argentinian is spelled 'Argentinean localization'. Both are correct ways to spell the adjective, but if you were multiple working on this, you might have spelled it differently within the module. Check among yourselves that you indeed used the same spelling throughout.

Hi @samueljlieber I think we should use this one 'Argentinean localization' as is the one that we have in the database and on the screenshots, as both are OK, let's keep the one we have been using when installing l10n_ar in a database, as it will more difficult to change the name there (and on the screenshots) than in this DOC.

Let me know what are your thoughts on this.
Thanks!
cc' @gza-odoo @fvz-odoo @ren-odoo @toaa-odoo

samueljlieber commented:
@vbe-odoo I think your suggestion is the way to go - I will update the document to use _"Argentinean localization"_. Thanks!

samueljlieber commented:
Commit a02776e to update `Argentinian` to `Argentinean` throughout the document 🙂

CC: @toaa-odoo

toaa-odoo reviewed (approved):
Great work, you won't hear from me again! :)

Tagging @jcs-odoo for final review.

StraubCreative commented:
[53b451c](https://github.com/odoo/documentation/pull/2430/commits/53b451c1b77a03ffd7722d73c929a8ed412c33f2) addressed merge conflict on prior commit [a02776e](https://github.com/odoo/documentation/commit/a02776e1ab4da6859fdda179316ea9eeff50deea) after #2983 went into effect.

Ready for your review with all checks passing @jcs-odoo

jcs-odoo reviewed (changes requested):
Apologies for this huge delay 🙈

I didn't go into detail for the whole doc but in general, I think this doc could refer more to the rest of the doc. Normally, we would add [doc links](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#documentation-pages-hyperlinks) or [custom anchors to specific headings](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#custom-anchors) in ["see also" sections](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#seealso).

Also, I think that there are too many images, but it's also okay like that for now. I saw that they were not [compressed with pngquant](https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#images) and that you forgot to delete the old images that are not useful anymore.

I already applied these changes and a few typos I caught here and there. I'll push them on this branch in a few minutes

jcs-odoo commented:
Pinging for merge already so we can move forward. We can still improve afterwards if needed :)

StraubCreative commented:
Thanks for the review @jcs-odoo 
It _seems_ like some of your suggestions would benefit from the new model/structure introduced in #2659.
So we'll push that once next and then take all of your feedback into a new PR once both are merged.

cc: @samueljlieber

AntoineVDV commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -73,567 +79,669 @@ Electronic Invoice Credentials
 
-Go to :menuselection:`Accounting --> Settings --> Argentinian Localization` to select the environment:
+To select a database environment, go to :menuselection:`Accounting --> Settings --> Argentinean
+Localization` and choose either :guilabel:`Prueba (Testing)` or :guilabel:`Produccion (Production)`
+.
```
```suggestion
Localization` and choose either :guilabel:`Prueba (Testing)` or :guilabel:`Produccion (Production)`.
```

AntoineVDV commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -73,567 +79,669 @@ Electronic Invoice Credentials
+   :alt: A list showing less common Argentinean tax options, which are labeled as inactive in Odoo
+         by default.
+
+.. _document-types:
```
This would conflict with any other `document-types` anchor added later in the doc, which is likely to happen.
```suggestion
.. _localizations/argentina/document-types:
```

AntoineVDV commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -73,567 +79,669 @@ Electronic Invoice Credentials
-Argentina case: AFIP).
+In some Latin American countries, like Argentina, some accounting transactions such as invoices and
+vendor bills are classified by document types defined by the governmental fiscal authorities. In
+Argentina, the `AFIP <https://www.afip.gob.ar/>`__ is the governmental fiscal authority that
```
Use single underscores in links.
```suggestion
Argentina, the `AFIP <https://www.afip.gob.ar/>`_ is the governmental fiscal authority that
```

AntoineVDV commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -73,567 +79,669 @@ Electronic Invoice Credentials
-- Condition applied based on the type of Issues and Receiver (ex. Type of fiscal regimen of
-  the buyer and type of fiscal regimen of the vendor)
+- The journal entry related to the invoice (if the journal uses documents);
+- The onditions applied based on the type of issuer and receiver (e.g., the type of fiscal regime of
```
```suggestion
- The conditions applied based on the type of issuer and receiver (e.g., the type of fiscal regime of
```

AntoineVDV commented on content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst:
```diff
@@ -73,567 +79,669 @@ Electronic Invoice Credentials
-   :align: center
+For auditing and troubleshooting purposes, it is possible to obtain detailed information of an
+invoice number that has been previously sent to the AFIP. To retrieve this information, go into
+:doc:`Developer Mode <../../../../general/developer_mode>`, then go to the
```
Either:
- use `doc` and start from the root to limit the number of `../`: `/applications/general/developer_mode`
- use `ref` (exceptionally) with the target `developer-mode`.

AntoineVDV reviewed (approved):
@robodoo r+
